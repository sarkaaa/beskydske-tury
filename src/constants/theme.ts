import * as colors from "./colors";
import * as sizes from "./sizes";

const theme = {
  colors,
  sizes
};

export default theme;