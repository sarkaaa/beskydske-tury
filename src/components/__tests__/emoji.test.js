import React from "react"
import renderer from "react-test-renderer"
import { ThemeProvider } from "styled-components"
import "jest-styled-components"

import Emoji from "../emoji"
import theme from "../../constants/theme"

const Wrapper = ({ children }) => (
  <ThemeProvider theme={theme}>{children}</ThemeProvider>
)

test("Emoji renders correctly", () => {
  const tree = renderer
    .create(
      <Wrapper>
        <Emoji label="label" symbol="🐻" emojiSize="large" />
      </Wrapper>
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})
