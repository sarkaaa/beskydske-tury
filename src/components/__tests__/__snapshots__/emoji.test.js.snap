// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Emoji renders correctly 1`] = `
.c0 {
  font-size: 10rem;
}

<span
  aria-hidden="false"
  aria-label="label"
  className="c0"
  role="img"
>
  🐻
</span>
`;
